classes/image_types_tegra.bbclass

```diff
@@ -295,7 +321,7 @@ create_tegraflash_pkg:tegra194() {
         cp "${IMAGE_TEGRAFLASH_DATA}" ./${DATAFILE}
         DATAARGS="--datafile ${DATAFILE}"
     fi
-    cp -L "${DEPLOY_DIR_IMAGE}/${DTBFILE}" ./${DTBFILE}
+    copy_dtbs "${WORKDIR}/tegraflash"
```

Comment:
Note that the resultant functionality in a non-default setup could differ slightly from the original. Normally, `DTBFILE` is generated as the first entry in `KERNEL_DEVICETREE`, and only one entry is in `KERNEL_DEVICETREE`, so there is no functional change. Since it may be useful to copy over multiple device tree binaries (overlay support for EFI) multiple entries in `KERNEL_DEVICETREE` or from a `virtual/dtb` provided will all be copied over.

Reply:
This will change the default setup, too, but I'm cool with that, since we actually have multiple DTBs listed for Orin.

However, we currently have separate `cp` commands for copying over all `.dtbo` files from `${DEPLOY_DIR_IMAGE}`, too. If you want this other-DTB-provider to cover the overlays as well, then you should probably change those commands, too.  It's a little tricky, since (by default) some of the overlays originate from the UEFI build, while some originate from the kernel build.

Reply:
> Is the concern that this is not doing enough to ship all the overlay content from the virtual/dtb provider?

No, I was concerned that if the intent is to have the `virtual/dtb` provider replace DTB/DTBO files that would normally come from either the kernel build or the UEFI build (or the NVIDIA prebuilts), to make sure that the other copy command  won't wipe out the ones being provided here.

Reply:
Ok, understood. What I just move the copy command down below? I can emit an info message when replacing existing content. It seems that anything generated in a custom recipe should be what the user intends to use.

Reply:
I wasn't sure whether just moving the `copy_dtbs` to after was the right way to deal with it, or skipping the default `*.dtbo` copy altogether if `EXTERNAL_KERNEL_DEVICETREE` is defined, I figured you'd know. :)

Reply:
It appears to be. I'll take the time to finish evaluating the switch from the plugin-manager to bootloader and kernel overlays for my custom hardware, just to be sure nothing is missed, and then make the call.

Reply:
I've moved the `copy_dtbs` below the other dtb copies, so that any user-generated content will get priority.